src/openvic-simulation/politics/Rebel.cpp

```diff
@@ -172,14 +177,15 @@ bool RebelManager::load_rebels_file(
 
 bool RebelManager::generate_modifiers(ModifierManager& modifier_manager) const {
 	bool ret = true;
+	const std::string identifier = "rebel_org_gain";
+	constexpr bool is_positive_good = false;
```

Comment:
```suggestion
	static constexpr std::string_view identifier = "rebel_org_gain";
	static constexpr bool is_positive_good = false;
```
String literals are part of the exe so their memory exists forever, no need to create a copy in a `std::string` when we can just reference the string literal memory with a `std::string_view`